### DISCUSSION TITLE
I can't find the ControNet options in the xyz script, can anyone give me ideas?

### DISCUSSION THREAD

**Discussion by sunhot223** [lllyasviel/stable-diffusion-webui-forge#499] [Q&A]
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/54968772/fe60bc4e-4f19-4748-a1ad-a14289b8c388)
just like the title, I can't find the contronet options.

**Comment by aiaisd** [comment]
I am in trouble too. I hope to improve.
